packages/hub/src/lib/common.ts

```diff
@@ -1,2 +1,13 @@
 // for format() from date-fns
 export const commonDateFormat = "MMM d yyyy, H:mm";
+
+export const DISCORD_URL = "https://discord.gg/nsTnQTgtG";
+export const GITHUB_URL = "https://github.com/quantified-uncertainty/squiggle";
+export const GITHUB_DISCUSSION_URL =
+  "https://github.com/quantified-uncertainty/squiggle/discussions";
+export const GRAPHQL_URL = "https://squigglehub.org/api/graphql";
+export const NEWSLETTER_URL = "https://quri.substack.com/t/squiggle";
+
+export const QURI_DONATE_URL = "https://quantifieduncertainty.org/donate";
+export const SQUIGGLE_DOCS_URL =
```

Comment:
Do you think I should move these to routes? Wasn't sure.

Reply:
GRAPHQL_URL - yes, everything else - probably not.

Reply:
About `GRAPHQL_URL`, we already have `graphqlPlaygroundRoute` in routes.

Reply:
That works, thanks!